components/automate-gateway/api/iam/v2/policy.proto

```diff
@@ -115,7 +208,16 @@ service Policies {
     option (chef.automate.api.policy).action = "update";
     option (chef.automate.api.iam.policy).resource = "iam:policies:{id}:members";
     option (chef.automate.api.iam.policy).action = "iam:policyMembers:update";
-    };
+    option (grpc.gateway.protoc_gen_swagger.options.openapiv2_operation) = {
+			tags: "policies";
+		};
+  };
+  /*
+  Remove policy members
+
+  Remove specific members from the member list for a specific policy. Silently ignores
+  members that are not already part of the member list.
```

Comment:
In addition to my comment above about format, also need this:
"The removed members will still exist within Chef Automate, but are no longer associated with this policy."